src/app/root.js

```diff
@@ -32,7 +31,7 @@ function rootReducer(state = {}, action) {
         draftState.visitURL = action.payload;
         break;
       case actionsList.logOut:
-        delete draftState.bakery.storage._store.identity;
+        delete bakery.storage._store.identity;
```

Comment:
I'm not sure why we are doing this at 2 places: here & `actions/logOut`?

Reply:
Ah yeah, well spotted. I can't see any reason for that so I'll remove it from here.